src/components/application_manager/src/app_launch/device_apps_launcher.cc

```diff
@@ -157,6 +158,23 @@ bool DeviceAppsLauncherImpl::StopLaunchingAppsOnDevice(
   return true;
 }
 
+void DeviceAppsLauncherImpl::StopLauncher(LauncherPtr launcher) {
+  SDL_LOG_AUTO_TRACE();
+  launcher->Clear();
+  free_launchers_.push_back(launcher);
+}
+
+void DeviceAppsLauncherImpl::StopLaunchingAppsOnAllDevices() {
+  SDL_LOG_AUTO_TRACE();
+  sync_primitives::AutoLock lock(launchers_lock_);
+
+  std::for_each(works_launchers_.begin(),
+                works_launchers_.end(),
+                boost::bind(&DeviceAppsLauncherImpl::StopLauncher, this, _1));
```

Comment:
@ychernysheva I think you can use bind from `std` library now

Reply:
Replaced with lambda in 792b6e0